src/settings.cpp

```diff
@@ -232,8 +232,6 @@ void CClientSettings::ReadSettingsFromXML ( const QDomDocument& IniXMLDocument,
     int  iValue;
     bool bValue;
 
-    bCleanUpLegacyFaderSettings = CommandLineOptions.contains ( "--cleanuplegacyfadersettings" );
```

Comment:
Is passing ComandLineOptions still needed?

Otherwiese we don't fully undo https://github.com/jamulussoftware/jamulus/pull/2839/files

Reply:
It's used in `ReadSettingsFromXML` in CSettings generally - it should be issuing an unused variable warning...  Hm.

Reply:
Are you sure? I don't find it in this function anymore.

Reply:
```
src/settings.cpp:36:    ReadSettingsFromXML ( IniXMLDocument, CommandLineOptions );
```